lib/puppet/util/filetype.rb

```diff
@@ -283,13 +307,21 @@ def write(text)
   newfiletype(:aixtab) do
     # Read a specific @path's cron tab.
     def read
+      unless Puppet::Util.uid(@path)
+        Puppet.debug _("The %{path} user does not exist. Treating their crontab file as empty in case Puppet creates them in the middle of the run.") % { path: @path }
+
+        return ""
+      end
+
       Puppet::Util::Execution.execute(%w{crontab -l}, cronargs)
     rescue => detail
       case detail.to_s
       when /open.*in.*directory/
         return ""
-      when /You are not authorized to use the cron command/
-        raise FileReadError, _("User %{path} not authorized to use cron") % { path: @path }, detail.backtrace
+      when /not.*authorized.*cron/
+        Puppet.err _("The %{path} user is not authorized to use cron. Their crontab file is treated as empty in case Puppet modifies e.g. cron.deny or cron.allow in the middle of the run.") % { path: @path }
```

Comment:
Docs guidelines suggest avoiding latin abbreviations like `e.g.` and using abbreviated English phrasing like "for example" instead.

Reply:
Thanks, I reworded the message.